dev/source/docs/simulation-2.rst

```diff
@@ -6,11 +6,26 @@ Simulation
 
 .. image:: ../images/simulation-landingpage.jpg
 
-Simulation allows for safe testing of experimental code and settings and crashing virtual vehicles is a lot cheaper than crashing real ones!
+Simulation allows for safe testing of experimental code and settings, and crashing virtual vehicles is a lot cheaper than crashing real ones!
+
+Overview
+========
+
+Simulation is implemented by using a Flight Dynamics Model (FDM) of the vehicle to simulate the physics involved with vehicle movement, which receives inputs from a SITL (Software in the Loop) program running the ArduPilot firmware (which are the firmware's servo/motor outputs) and outputs vehicle status,position, velocities,etc. that result from those inputs back to the firmware sim just like sensors would in the real world.
+
+Pilot control is implemented either by joystick/sim controller box or by MAVLink commands from an attached and monitoring Ground Control Station program (GCS), like MAVProxy or Mission Planner.
+
+This is illustrated below:
+
+.. image:: ../images/SITL.png
+
+ArduPilot provides a native firmware simulator + FDM program (sim_vehicle.py) + graphical visualizer, simply referred to as ``SITL``, which is usually used with its developer GCS, Mavproxy.
```

Comment:
```suggestion
ArduPilot provides a native firmware simulator + FDM program (sim_vehicle.py) + graphical visualizer, often simply referred to as ``SITL``, which is usually used with its developer GCS, Mavproxy.
```

Sometimes the MissionPlanner use of SITL is just referred to as "SITL" too :+1:

Reply:
reworded and expanded